data/en/construction_0001.json

```diff
@@ -1117,7 +1088,7 @@
                             },
                             {
                                 "goto": {
-                                    "group": "group683013fe-88a1-47c3-975c-fbf36c3ec625"
+                                    "group": "groupe9d26530-ad8f-4527-b09c-afacfa0c820a"
```

Comment:
Should this be spelled `groupe`? Or is that part of the ID? Not sure if they always start with a number or not.

Reply:
Can start with a number or a letter. If it has to start with a number it will be less random

Reply:
It's part of the ID